config/metrics.json

```
@@ -10,6 +10,7 @@
 			"uniqueItems": true
 		},
 		"AllowsDeletions": { "type": ["boolean", "null"] },
+		"AllowsCocs": { "type": ["boolean", "null"] },
```

if "AllowsCocs" is a name we choose, let's use this one:
```suggestion
		"HasCodeOfConduct": { "type": ["boolean", "null"] },
```